code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -1,6 +1,5 @@
 /mob/living/simple_animal/hostile
 	faction = list("hostile")
-	mouse_opacity = 2 //This makes it easier to hit hostile mobs, you only need to click on their tile, and is set back to 1 when they die

```

Comment:
This was removed completely. It made it impossible to play as megafauna (who have gigantic sprites) as you'd constantly attack yourself with very high damage attacks that would kill you in a few hits.